(.: ((((sheep --> (IntSet see)) &/ (^ count)) =/> (sheep --> (IntSet counted))) (1.0 0.9)))
thread '<unnamed>' panicked at lib/src/metta/types.rs:142:22:
Incorrect function type: (->)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at core/src/panicking.rs:221:5:
panic in a function that cannot unwind
stack backtrace:
   0:     0x7f5bd95ee0aa - <std::sys::backtrace::BacktraceLock::print::DisplayBacktrace as core::fmt::Display>::fmt::h304520fd6a30aa07
   1:     0x7f5bd963b5eb - core::fmt::write::hf5713710ce10ff22
   2:     0x7f5bd95e26a3 - <unknown>
   3:     0x7f5bd95f08a2 - <unknown>
   4:     0x7f5bd95f050e - std::panicking::default_hook::h81c8cd2e7c59ee33
   5:     0x7f5bd95f11ef - std::panicking::rust_panic_with_hook::had2118629c312a4a
   6:     0x7f5bd95f0ea3 - <unknown>
   7:     0x7f5bd95ee589 - <unknown>
   8:     0x7f5bd95f0b64 - rust_begin_unwind
   9:     0x7f5bd9209935 - core::panicking::panic_nounwind_fmt::hc0ae93930ea8f76c
  10:     0x7f5bd92099c2 - core::panicking::panic_nounwind::h9f485ff9b02bac75
  11:     0x7f5bd9209be6 - core::panicking::panic_cannot_unwind::hea865182d7ce50af
  12:     0x7f5bd924d962 - metta_run
  13:     0x7f5bd922cf7c - <unknown>
  14:     0x7f5bd92143ef - <unknown>
  15:     0x55b1255d2282 - <unknown>
  16:     0x55b1255c8b4b - _PyObject_MakeTpCall
  17:     0x55b1255c24c8 - _PyEval_EvalFrameDefault
  18:     0x55b1255d2aec - _PyFunction_Vectorcall
  19:     0x55b1255bcae8 - _PyEval_EvalFrameDefault
  20:     0x55b1255d2aec - _PyFunction_Vectorcall
  21:     0x55b1255bc9a2 - _PyEval_EvalFrameDefault
  22:     0x55b1256a1e56 - <unknown>
  23:     0x55b1256a1d26 - PyEval_EvalCode
  24:     0x55b1256c8ae8 - <unknown>
  25:     0x55b1256c32ef - <unknown>
  26:     0x55b1256c8885 - <unknown>
  27:     0x55b1256c7e68 - _PyRun_SimpleFileObject
  28:     0x55b1256c7b47 - _PyRun_AnyFileObject
  29:     0x55b1256bc02e - Py_RunMain
  30:     0x55b125695d6d - Py_BytesMain
  31:     0x7f5bda0b7d90 - <unknown>
  32:     0x7f5bda0b7e40 - __libc_start_main
  33:     0x55b125695c65 - _start
  34:                0x0 - <unknown>
thread caused non-unwinding panic. aborting.
timeout: the monitored command dumped core
real 0.37
user 0.25
sys 0.00
